packages/@glimmer/runtime/lib/helpers/get.ts

```diff
@@ -88,21 +89,17 @@ export default internalHelper((args: VMArguments) => {
     () => {
       let source = valueForRef(sourceRef);
 
-      if (isObject(source)) {
+      if (isDict(source)) {
```

Comment:
For the lazy, this is doing `source !== null && source !== undefined`

https://github.com/glimmerjs/glimmer-vm/blob/1a54cba8a6b4175cb2d5a617ff857b616cf6f0b7/packages/%40glimmer/util/lib/collections.ts#L13-L15